x/tx/CHANGELOG.md

```diff
@@ -31,13 +31,16 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## v0.9.0
```

Comment:
I'll be tagging v0.9.0 of `x/tx` after this PR.

Reply:
Ah, I'll probably wait for this to be fixed before tagging: https://github.com/cosmos/cosmos-sdk/pull/16521#issuecomment-1614507574 forgot about this.